File: Python/HumanBody.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from Cell import Cell


class HumanBody(object):
	def get_cell(self):
		"""@ReturnType Cell"""
		return self.___cell

	def set_cell(self, a_cell):
		"""@ParamType aCell Cell
		@ReturnType void"""
		self.___cell = a_cell

	def __init__(self, a_cell):
		self.___cell = a_cell
		"""@AttributeType Cell"""
		self._theCell = []
		"""@AttributeType Cell*
		# @AssociationType Cell[]
		# @AssociationMultiplicity *
		# @AssociationKind Composition"""

	def to_string(self):
		a = self.get_cell()
		Cell.to_s(a)

File: JAVA/HumanBody.java
public class HumanBody {

	private Cell cell;

	public HumanBody(Cell cell1) {
        cell = cell1;
    }

    public Cell getCell() {
		return this.cell;
	}

	public void setCell(Cell cell) {
		this.cell = cell;
	}

    public void to_String(){
        cell.to_String();
    }
}